packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -1054,6 +1045,8 @@ class ThemeData with Diagnosticable {
   final bool fixTextFieldOutlineLabel;
 
   /// Creates a copy of this theme but with the given fields replaced with the new values.
+  ///
+  /// The [brightness] value is applied to the [colorScheme].
```

Comment:
This seems a little odd. When would you want to change the brightness of the colorScheme without changing its colors as well? Is this because we don't want to break the existing API by removing the brightness param here?

Reply:
Yes. I found one use-case in an internal app so I'm assuming that there are others in outside apps.

I think this is comparable to allowing the ThemeData brightness constructor parameter override the color scheme's brightness.